qa/L0_backend_python/common.sh

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
So now inside multiple test scripts? For ensemble, environment and lifecycle. And we keep the generic tests in python_test.py? 
Why not have a single test.sh with all the above?

Reply:
Yes, that is correct. Because  `test.sh` becomes very large and hard to maintain.